### ISSUE TITLE
Using pdb.set_trace() blocks

### ISSUE THREAD

**Issue by notoriousno** [williamsjj/shiji#2]
While debugging, I noticed when I place a `pdb.set_trace()` in a render function (eg: `render_GET`), the entire application stops.  However, it doesn't seem like the main Twisted part isn't running in background thread, so I'm not sure why this occurs.  Is `ReactorInspector` or the statd thread to blame?

PS
Great project!

**Comment by williamsjj** [comment]
Twisted apps are single threaded, so anything that pauses execution will block the entire program until it resumes.

**Comment by notoriousno** [comment]
Ahhh so sorry I should've been more descriptive in my original post.  Yes I'm aware Twisted is single threaded.  I placed a `pdb.set_trace()` so that I could step through and interact with code via `pdb`.  However, when I hit the break point no `pdb` console appears.  In my experience, this usually occurs when there are multiple threads involved, which is why I asked if `ReactorInspector` or statd thread was interfering.  In any case, I ended up "debugging" by using `twisted.python.log` to write the values I want to look at, but using `pdb` would be more ideal in the future.
